dfir_lang/src/graph/flat_to_partitioned.rs

```diff
@@ -371,9 +375,14 @@ fn find_subgraph_strata(
         |u| subgraph_graph.succs.get(&u).into_iter().flatten().cloned(),
     );
 
-    // Each subgraph's stratum number is the same as it's predecessors. Unless there is a negative
-    // edge, then we increment.
+    // Each subgraph's stratum number is the same as it's predecessors.
+    //
+    // Unless:
+    // - At the top level: there is a negative edge (e.g. `fold()`), then we increment.
```

Comment:
Why are strata special at the top level? Is this a terminology confusion ("strata" is for datalog, but you're overloading it for loop counting) or a single concept that is different in context at the top level? I'm a little uncertain what's intended. I realize this may be more a design question than a code question, but design clarity might help with variable naming and comments and vice versa.

Reply:
It's a relic to preserve backwards compatibility until we remove top-level barriers